java/src/main/native/OrtJniUtil.h

```diff
@@ -92,6 +94,24 @@ jsize safecast_size_t_to_jsize(size_t v);
 
 jsize safecast_int64_to_jsize(int64_t v);
 
+#ifdef _WIN32
+#include <Intsafe.h>
+static inline void* allocarray(size_t nmemb, size_t size) {
```

Comment:
>static [](http://example.com/codeflow?start=0&length=6)

This presumably compiles these function duplicates into each of the .c files.

Reply:
Only when the function is used. And it is only used in two files.  And this function has "inline" keyword. So it will be duplicated every time it is used, not one per each *.c file. However, I don't think it would have much impact on binary size as this function is quite short,  For example, on Linux it just directly calls reallocarray.